homeassistant/components/sleepiq/entity.py

```diff
@@ -39,5 +42,3 @@ def _handle_coordinator_update(self) -> None:
     @callback
     def _async_update_attrs(self) -> None:
         """Update sensor attributes."""
```

Comment:
If there isn't every going to be anything to call here, you could decorate this with `@abstractmethod` and remove the `super()` in the classes that use this as a base class